src/test/java/api/support/matchers/JsonObjectMatcher.java

```diff
@@ -50,6 +52,28 @@ public void describeTo(Description description) {
     };
   }
 
+  public static Matcher<JsonObject> hasNoJsonPath(String jsonPath) {
```

Comment:
Does this check that an JSON object does not have a sub-object at that path?

Reply:
Yes, it actually delegates logic to jayway matcher.

Reply:
Ok, thanks. I was considering giving it a more explicit name, however I think keeping it the same as the jayway matcher is a good idea